halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/GateProfileFactory.java

```diff
@@ -60,17 +60,11 @@ public void setProfile(Profile profile,
     SpringConfig spring;
     SamlConfig saml;
     LdapConfig ldap;
+    X509Config x509;
 
     GateConfig(ServiceSettings gate, Security security) {
       super(gate);
       server.ssl = security.getApiSecurity().getSsl();
-      if (security.getAuthn().getOauth2().isEnabled()) {
```

Comment:
I assume these are taken care of in the two profile factories instead of the original one.